sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/CosmosContainerTest.java

```diff
@@ -264,6 +258,40 @@ public void readAllContainers() throws Exception{
         assertThat(feedResponseIterator1.iterator().hasNext()).isTrue();
     }
 
+    @Test(groups = { "emulator" }, timeOut = TIMEOUT)
+    public void crudMultiHashContainerTest() throws Exception{
```

Comment:
code style:
we don't use "Test" suffix nor prefix. please remove the Test suffix.

```suggestion
    public void crudMultiHashContainer() throws Exception {
```